ray-operator/controllers/ray/raycluster_controller_test.go

```diff
@@ -266,17 +261,18 @@ var _ = Context("Inside the default namespace", func() {
 
 		It("should update a raycluster object", func() {
 			// adding a scale strategy
-			Eventually(
-				getResourceFunc(ctx, client.ObjectKey{Name: myRayCluster.Name, Namespace: "default"}, myRayCluster),
-				time.Second*9, time.Millisecond*500).Should(BeNil(), "My raycluster = %v", myRayCluster)
-
-			podToDelete1 := workerPods.Items[0]
-			rep := new(int32)
-			*rep = 1
-			myRayCluster.Spec.WorkerGroupSpecs[0].Replicas = rep
-			myRayCluster.Spec.WorkerGroupSpecs[0].ScaleStrategy.WorkersToDelete = []string{podToDelete1.Name}
-
-			Expect(k8sClient.Update(ctx, myRayCluster)).Should(Succeed(), "failed to update test RayCluster resource")
+			err := retry.RetryOnConflict(retry.DefaultRetry, func() error {
+				Eventually(
+					getResourceFunc(ctx, client.ObjectKey{Name: myRayCluster.Name, Namespace: "default"}, myRayCluster),
+					time.Second*9, time.Millisecond*500).Should(BeNil(), "My raycluster = %v", myRayCluster)
+				podToDelete1 := workerPods.Items[0]
```

Comment:
```suggestion
				podToDelete := workerPods.Items[0]
```

I will expect to see `podToDelete2` below if the variable name is `podToDelete1` here.